__tests__/endpoints/app.test.js

```diff
@@ -130,6 +130,27 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe("GET /api/articles?topic=paper", () => {
+  test("200 and returns an array with paper topic articles", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(1);
+        expect(articles[0].topic).toBe("cats");
+      });
+  });
+
+  test("404 and returns Not found when there is not articles for a determined topic", () => {
```

Comment:
lets return a 200 with this provided the topic exists but there are no articles. If the topic doesn't exist then lets return a 404 😄